docs/block-api.md

```diff
@@ -105,6 +105,18 @@ By default, Gutenberg adds a class with the form `.wp-blocks-your-block-name` to
 className: false,
 ```
 
+### useOnce (optional)
+
+* **Type:** `Bool`
+* **Default:** `false`
```

Comment:
I'm not sure if it is explicitly defined as `false` in the code, but this is how it works :)